tracer/src/Datadog.Trace/IAST/Settings/IastSettings.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="IastSettings.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using Datadog.Trace.AppSec;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.Iast.Settings
+{
+    internal class IastSettings
+    {
+        public static readonly string WeakHashAlgorithmsDefault = "HMACMD5,MD5,HMACSHA1,SHA1";
+
+        public IastSettings(IConfigurationSource source)
+        {
+            WeakHashAlgorithms = source?.GetString(ConfigurationKeys.Iast.WeakHashAlgorithms) ?? WeakHashAlgorithmsDefault;
+            WeakHashAlgorithmsArray = WeakHashAlgorithms.Split(new char[] { ',' }, System.StringSplitOptions.RemoveEmptyEntries);
+            Enabled = (source?.GetBool(ConfigurationKeys.Iast.Enabled) ?? false) && WeakHashAlgorithmsArray.Length > 0;
```

Comment:
Not a blocker, but I find `== true` easier to read than `?? false`. Level of care low.
 
```suggestion
            Enabled = source?.GetBool(ConfigurationKeys.Iast.Enabled == true &&
                      WeakHashAlgorithmsArray.Length > 0;
```

Reply:
Done!